platform/android/java/lib/src/org/godotengine/godot/GodotLib.java

```diff
@@ -99,10 +99,20 @@
 	 */
 	public static native void hover(int type, int x, int y);
 
+	/**
+	 * Forward mouse pressed events from the main thread to the GL thread.
+	 */
+	public static native void mouse_pressed(int button, int x, int y, boolean pressed);
+
+	/**
+	 * Forward  mouse moved while pressed events from the main thread to the GL thread.
+	 */
+	public static native void mouse_moved_pressed(int button, int x, int y);
```

Comment:
This is an inconsistent order with `doubletap` (which could maybe be `double_tap` instead).

Reply:
what do you mean by inconsistent? I did click -> drag -> doubletap for all my changes.
I did it beacause I saw that hover is also not a touch event

Reply:
`mouse_moved_pressed(int button, int x, int y);`

`doubletap(int x, int y, int button);`

One is "button, x, y", the other is "x, y, button".